README.md

```diff
@@ -61,6 +61,37 @@ Or check out the latest [release](https://github.com/tharsis/evmos/releases).
 
 To learn how the Evmos works from a high-level perspective, go to the [Introduction](https://evmos.dev/intro/overview.html) section from the documentation. You can also check the instructions to [Run a Node](https://evmos.dev/quickstart/run_node.html).
 
+## Recording Transactions Per Second (TPS)
```

Comment:
thanks for the docs! can you move this section to `docs/quickstart/run_node.md`? 🙏

Reply:
Sounds good! Let me do so right now.

Reply:
Done!